cdap-ui/server/config/development/cdap.json

```diff
@@ -110,7 +110,7 @@
   "app.exec.threads": "20",
   "stream.bind.address": "127.0.0.1",
   "data.tx.snapshot.dir": "/cdap/tx.snapshot",
-  "dashboard.bind.port": "9999",
```

Comment:
This should not be in the PR right? We now have removed the old UI. So 9999 port should be available for us.